src/components/Navigation.js

```diff
@@ -7,7 +7,13 @@ import SearchBar from './Search';
 
 function Navigation() {
   return (
-    <Navbar collapseOnSelect expand='lg' bg='light' variant='light'>
+    <Navbar
+      collapseOnSelect
+      expand='lg'
+      bg='light'
+      variant='light'
+      style={{ paddingLeft: '30px', paddingRight: '30px', textAlign: 'center' }}
```

Comment:
I'm not sure if we want to center align them. Adding some extra padding to the left could do the job.

Reply:
Done!